cmd/zb/helper.go

```diff
@@ -0,0 +1,594 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path"
+	"time"
+
+	"github.com/google/uuid"
+	imeta "github.com/opencontainers/image-spec/specs-go"
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"gopkg.in/resty.v1"
+	"zotregistry.io/zot/errors"
+	"zotregistry.io/zot/pkg/test"
+)
+
+func deleteTestRepo(repos []string, url string, client *resty.Client) error {
+	for _, repo := range repos {
+		resp, err := client.R().Delete((fmt.Sprintf("%s/v2/%s/", url, repo)))
+		if err != nil {
+			return err
+		}
+
+		// request specific check
+		statusCode := resp.StatusCode()
+		if statusCode != http.StatusAccepted {
+			return errors.ErrUnknownCode
+		}
+	}
+
+	return nil
+}
+
+func pushImage(workdir, url, trepo string, repos []string, size int,
```

Comment:
There are two types of push we are testing right? Which one is this? Monolith or Chunk?